src/Lykke.Job.BlockchainCashinDetector.Core/Domain/IDepositBalanceDetectionsDeduplicationLock.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Lykke.Job.BlockchainCashinDetector.Core.Domain
+{
+    public interface IDepositBalanceDetectionsDeduplicationLock
```

Comment:
I believe that domain layer entity, repository entity, contract entity shouldn't implement the same interface, since each layer could represent entity in more suitable and comfortable for them way. So, to prevent propogation of the entity with the same interface all over the layers, this interface should be removed. Feel free to use automaper or analog to fill entities in cross-layer transitions.

And btw, let's do domain entities, not a DTO. Make fields, which are immutable by business, read only properties, add ctor, add method wich change entity state. This looks like overkill at the moment, but this entity could be extended and will be good if it became our pattern to implement domain entities with proper DSL but not just DTO.